tests/testsuite/credential_process.rs

```diff
@@ -94,6 +94,16 @@ fn warn_both_token_and_process() {
         .file("src/lib.rs", "")
         .build();
 
+    // HACK: Inject `foo` directly into the index so `publish` won't block for it to be in
```

Comment:
Or if we had a `--no-wait` option to `cargo publish` that could be used here.

Reply:
Thanks for calling that out; I had forgotten to include it in my list.

I had considered it but it would require "nightly" until #11222 is stablized and was wanting to keep the tests focused on "stable" for now.

I had also considered making the default be determined by the source so a local registry could always specify `0` but that  was going to require plumbing a fairly specific use case through several traits and I was hesitant on whether that was worth it.